keras_cv/bounding_box/converters.py

```diff
@@ -112,54 +112,43 @@ def decode_single_level(anchor, box_delta):
 
 
 def _center_yxhw_to_xyxy(boxes, images=None, image_shape=None):
-    y, x, height, width, rest = tf.split(
-        boxes, [1, 1, 1, 1, boxes.shape[-1] - 4], axis=-1
-    )
+    y, x, height, width = tf.split(boxes, [1, 1, 1, 1], axis=-1)
```

Comment:
nit: the original implement should handle both dim=4 and dim>4, so it's not strictly required to make this before 0.4, I guess.
But leaving it up to you